Gopkg.lock

```diff
@@ -419,6 +426,18 @@
   packages = ["."]
   revision = "5a0325d7fafaac12dda6e7fb8bd222ec1b69875e"
 
+[[projects]]
```

Comment:
why not in Gopkg.toml?

Reply:
@dipak-pawar Ah, thanks, will add it there.